modules/javafx.controls/src/main/java/javafx/scene/control/TableView.java

```diff
@@ -395,6 +395,92 @@
         }
     };
 
+    /**
+     * A policy that tries to adjust other columns in order to fit the table width.
+     * <p>
+     * During UI adjustment, proportionately resizes all columns.
+     * <p>
+     * When column constraints make it impossible to fit all the columns into the allowed area,
+     * the columns are either clipped, or an empty space appears.  This policy disables the horizontal
+     * scroll bar.
+     *
+     * @since 20
+     */
+    public static final Callback<TableView.ResizeFeatures, Boolean> CONSTRAINED_RESIZE_POLICY_ALL_COLUMNS =
+        ConstrainedColumnResize.forTable(ConstrainedColumnResize.ResizeMode.AUTO_RESIZE_ALL_COLUMNS);
+
+    /**
+     * A policy that tries to adjust last column in order to fit the table width.
+     * <p>
+     * During UI adjustment, resizes the last column only.
```

Comment:
You might want to add `to preserve the total width` here since you say it for some of the other policies.